docs/CLI.md

```diff
@@ -28,6 +28,18 @@ Run tests related to changed files based on hg/git (uncommitted files):
 jest -o
 ```
 
+Run tests in a file in a random order (this will print the seed used for random number generation):
```

Comment:
I get the idea, but what if this would be moved to `--randomize`? If a user is interested to read about `--randomize`, current text does not provide usage details. It is not really clear where to look for it and the details here are important.

Reply:
I've copied what I've written here to be part of the CLI description of the `--randomize` option.
I've done the same with `--seed` as well.

Was this what you were looking for?

Reply:
Thanks. I was trying to say that this usage information could be more useful, if placed next to `--randomize` description. That’s there I would expect to find it, but this is just an opinion (;

Reply:
ahh gotcha. Yeah that makes sense.
I initially put a description near the beginning because this might be a popular features. On further thinking it's probably not that popular.